collections/bitcoin/_posts/2022-10-02-bitcoin-is-digital-scarcity.md

```diff
@@ -713,7 +713,7 @@ disappear.
 [^fn-diamonds]: Diamonds are one example of a real-world good that used to be scarce but can now be farmed. Thanks to human ingenuity, diamonds can be produced artificially. To quote the [U.S. National Minerals Information Center](https://web.archive.org/web/20221002111535/https://www.usgs.gov/centers/national-minerals-information-center/industrial-diamond-statistics-and-information): "Synthetic industrial is superior to its natural diamond counterpart because it can be produced in unlimited quantities, and, in many cases, its properties can be tailored for specific applications. Consequently, manufactured diamond accounts for more than 90% of the industrial diamond used in the United States."
 
 When it comes to in-game currencies, the company that is running the game
-servers might go bankrupt or decides to shut down operations for other reasons.
+servers might go bankrupt or decide to shut down operations for other reasons.
```

Comment:
```suggestion
servers might go bankrupt or decides to shut down operations for other reasons.
```

The company [...] decides to shut down ...

Reply:
I respectfully disagree, it's the "might" in the sentence that changes it: The company might [go bankrupt or] decide to shut down operations